src/main/java/it/smartcommunitylab/aac/config/SecurityConfig.java

```diff
@@ -202,6 +211,7 @@ public void configure(HttpSecurity http) throws Exception {
                 .antMatchers("/img/**").permitAll()
                 .antMatchers("/italia/**").permitAll()
                 .antMatchers("/spid/**").permitAll()
+                .antMatchers("/webauthn/**").permitAll()
```

Comment:
what lives under /webauthn which should be accessible without authentication?

Reply:
Under `/webauthn/` we have the html, css and js files for the frontend and the routes of the WebAuthn pages/API.
At the moment all these resources are needed for the un-authenticated user in order to register and login, so I think that they should all be accessible without authentication.
Should I add more granular rules?

Reply:
All controllers and filters do live under /auth/webauthn